### DISCUSSION TITLE
Support for external download manager.

### DISCUSSION THREAD

**Discussion by nikhilbadyal** [bromite/bromite#949] [Ideas]
Download manager managers like IDM really provides lot of feature which are really needed. Any plan to integrate it ? 

PS - Even firefox support external manager.

**Comment by ghost** [comment]
"Only features related to privacy are considered"

**Comment by chromer030** [comment]
You can use feature of 'interception of links from android browsers and clipboard' in great download managers like ADM or IDM.
Integrating download managers in Chromium source needs lots of heavy modifications which easily could be broken after major updates.
